src/cmd/compile/README.md

```diff
@@ -79,8 +79,7 @@ historical reasons, but the long-term plan is to move all of them here.
 
 Then, a series of machine-independent passes and rules are applied. These do not
 concern any single computer architecture, and thus run on all `GOARCH` variants.
-
-Some examples of these generic passes include dead code elimination, removal of
+These passes include dead code elimination, removal of
```

Comment:
The term "generic" is used a few times in this document, which could be confusing given the work on generics as a language feature.